test/source/tests/setup.ts

```diff
@@ -436,6 +446,60 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await gmailPage.notPresent(['@webmail-notification']);
     }));
 
+    ava.default('mail.google.com - success notif after setup, click hides it, does not re-appear + offers to reauth', testWithBrowser('compatibility', async (t, browser) => {
+      const acct = 'flowcrypt.compatibility@gmail.com';
+      const gmailPage = await browser.newPage(t, TestUrls.mockGmailUrl(), undefined, await getAuthorizationHeader(t, browser, acct));
+      await gmailPage.waitAll(['@webmail-notification', '@notification-successfully-setup-action-close']);
+      await gmailPage.waitAndClick('@notification-successfully-setup-action-close', { confirmGone: true });
+      await gmailPage.page.reload();
+      await gmailPage.notPresent(['@webmail-notification', '@notification-setup-action-close', '@notification-successfully-setup-action-close']);
+      // below test that can re-auth after lost access (simulating situation when user changed password on google)
+      await Util.wipeGoogleTokensUsingExperimentalSettingsPage(t, browser, acct);
+      const settingsPage = await browser.newPage(t, TestUrls.extensionSettings(acct));
+      await settingsPage.waitAndRespondToModal('confirm', 'cancel', 'FlowCrypt must be re-connected to your Google account.');
+      // *** these tests below are very flaky in CI environment, Google will want to re-authenticate the user for whatever reason
+      // // opening secure compose should trigger an api call which causes a reconnect notification
+      await gmailPage.page.reload();
+      await gmailPage.waitAndClick('@action-secure-compose');
+      await gmailPage.waitAll(['@webmail-notification', '@action-reconnect-account']);
+      await Util.sleep(1);
+      expect(await gmailPage.read('@webmail-notification')).to.contain('Please reconnect FlowCrypt to your Gmail Account.');
+      const oauthPopup = await browser.newPageTriggeredBy(t, () => gmailPage.waitAndClick('@action-reconnect-account'));
+      await OauthPageRecipe.google(t, oauthPopup, acct, 'approve');
+      await gmailPage.waitAll(['@webmail-notification']);
+      await Util.sleep(1);
+      expect(await gmailPage.read('@webmail-notification')).to.contain('Connected successfully. You may need to reload the tab.');
+      // reload and test that it has no more notifications
+      await gmailPage.page.reload();
+      await gmailPage.waitAndClick('@action-secure-compose');
+      await Util.sleep(1);
+      await gmailPage.notPresent(['@webmail-notification']);
+    }));
+
+    ava.default('mail.google.com - setup prompt notif + hides when close clicked + reappears + setup link opens settings', testWithBrowser(undefined, async (t, browser) => {
+      const acct = 'flowcrypt.compatibility@gmail.com';
+      const settingsPage = await openMockGmailPage(t, browser, acct);
```

Comment:
What are lines 481 and 482 actually needed?

Reply:
It's indeed not needed